frontend/device/android/device.lua

```diff
@@ -90,6 +90,10 @@ function Device:initNetworkManager(NetworkMgr)
     end
 end
 
+function Device:networkInfo()
+    return android.getNetworkInfo()
```

Comment:
Hehe, this. But anyway, like I said [there](https://github.com/koreader/android-luajit-launcher/pull/104#discussion_r253885562) I think this is where any potential string formatting should occur.

Reply:
And I think that'd make it the Android version of retrieveNetworkInfo().